package.json

```diff
@@ -47,6 +47,8 @@
     "api-documenter": "api-documenter yaml --input-folder=temp"
   },
   "dependencies": {
+    "@google-cloud/paginator": "^3.0.0",
```

Comment:
`@google-cloud/paginator` was explicitly removed here:
https://github.com/googleapis/nodejs-bigtable/commit/8a2a74e00c1241c89bf344db0b82440577270223

@AVaksman want to work with @zamnuts on what we're supposed to be doing here now?

Reply:
> `@google-cloud/paginator` was explicitly removed here:
> https://github.com/googleapis/nodejs-bigtable/commit/8a2a74e00c1241c89bf344db0b82440577270223
> 
> @AVaksman want to work with @zamnuts on what we're supposed to be doing here now?

I noticed that commit. I've successfully refactored the stream portion, but am still having trouble understanding the pagination.

Reply:
Sure,

- add gapic method name (`listBackupsStream: true`) to [`gapicStreamingMethods`](https://github.com/googleapis/nodejs-bigtable/blob/2917d177fe2a69bb7cd0f652bd3318aee359dd4d/src/index.ts#L753)
- pipe, rather `pumpify` received objects through a `Transform` stream that will convert proto objects into this library `Backup` objects. Similar reference below.

https://github.com/googleapis/nodejs-bigtable/blob/2917d177fe2a69bb7cd0f652bd3318aee359dd4d/src/instance.ts#L727-L744